modules/vba_documents/spec/jobs/upload_remover_spec.rb

```diff
@@ -17,8 +17,8 @@
   end
 
   describe '#perform' do
-    describe 'when the record is older than 3 days' do
-      let(:upload) { FactoryBot.create(:upload_submission, status: 'received', created_at: Time.zone.now - 4.days) }
+    describe 'when the record is older than 10 days' do
+      let(:upload) { FactoryBot.create(:upload_submission, status: 'received', created_at: Time.zone.now - 11.days) }
```

Comment:
How do you feel about `11.days.ago` to make it more readable?

Reply:
Sounds good, changed thanks.